processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -88,6 +90,9 @@
       .put("Long", ValueType.LONG)
       .put("Double", ValueType.FLOAT)
       .put("String", ValueType.STRING)
+      .put("LONG", ValueType.LONG)
+      .put("FLOAT", ValueType.FLOAT)
+      .put("STRING", ValueType.STRING)
```

Comment:
@fjy Changed this to an <Object, ValueType> map with comments explaining the use cases